tox.ini

```diff
@@ -1,10 +1,13 @@
 [tox]
 envlist=
-    py{26,27,33,34,35,36}-{requests-,}test,
+    py{26,27,33,34,35,36,37,38}-test,
+    py{26,27,33,34,35,36,37,38}-requests,
     py{26,27}-flask,
-    py{26,27,33,34,35,36}-wsgi,
-    py{27,36}-django{18,19,110,111,20,21}-migrate1-django1
-    py36-lint,
+    py{26,27,33,34,35,36,37,38}-wsgi,
+    py{35,36,37}-django{18,19,110,111,20,21}-migrate1-django1,
+    py27-django{18,19,110,111}-migrate1-django1,
+    py38-django{18,111,20,21}-migrate1-django1,
+    py38-lint
```

Comment:
I've split out these to remove combinations that aren't valid, e.g. `py27-django20-migrate1-django1` doesn't work so it's not longer here